client/components/accordion.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import ListGenerator from './list-generator';
+
+function ContentList(props) {
+  return (
+    <div className="ac-content">
+      <ol>
+        <ListGenerator content={ props.content } />
+      </ol>
+    </div>
+  );
+}
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      verifyID: null
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ isLoading: false });
+  }
+
+  handleClick(id) {
+    const targetID = id;
+    const { verifyID } = this.state;
+    verifyID === targetID
+      ? this.setState({ verifyID: null })
+      : this.setState({ verifyID: targetID });
+  }
+
+  render() {
+    const { verifyID, isLoading } = this.state;
+    if (isLoading) {
+      return <h4>Loading...</h4>;
+    }
+    const { data } = this.props;
+    const renderData = data.map(index => {
```

Comment:
`data` is usually considered a bad variable name. Data about what? try renaming your prop, cuz idk what this is supposed to be.